common/src/config.rs

```diff
@@ -285,7 +289,9 @@ impl Default for Limits {
         Limits {
             max_concurrent_requests: 100,
             max_concurrent_queries: 5,
-            max_event_payload_size: ByteSize::from_kilobytes(256),
+            max_plain_event_payload_size: ByteSize::from_kilobytes(256),
+            max_security_event_payload_size: ByteSize::from_kilobytes(256),
+            max_minidump_event_payload_size: ByteSize::from_megabytes(2000),
```

Comment:
I see a lot of people hosting on-prem want to raise size limits because they don't care if their installation is on fire from a single event. Based on that I thought we could set the defaults to cater to the onprem usecase rather than ours.

In some sense hosting a relay for multiple organizations (our own usecase) is an edgecase.

Does that make sense? I have no context about minidumps in particular and don't know if people are likely to send minidumps this large. I just know it from event payload trimming.

Reply:
ok